docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -97,6 +97,36 @@ You can also use `winget`, Microsoft's Package manager for Windows, to install a
 
 The `-e` option is to ensure the official Azure CLI package is installed. This command installs the latest version by default. To specify a version, add a `-v <version>` with your desired version to the command.
 
+# [ZIP Package](#tab/zip)
+
+> [!IMPORTANT]
+> This package is currently in preview.
+
+You can use the ZIP package to install the Azure CLI on Windows. This package is useful when you don't have administrative privilege.
+
+Unzip the package to a folder, then using Cmd or PowerShell, call the Azure CLI by running `<unzipped folder path>\bin\az.cmd`.
```

Comment:
Same as https://github.com/MicrosoftDocs/azure-docs-cli/pull/4045#discussion_r1473156429, I suggest we do mention how to call it if the path contains space - in both CMD and PowerShell.

Reply:
I agree with @jiasli.  I don't feel it is possible to have too much clarity.  Is is possible to have too many words!  :-)

Our install docs receive the most negative feedback of any doc group in our repo, so I prefer not to wait for a customer complaint.